package/MDAnalysis/core/groups.py

```diff
@@ -1520,6 +1520,8 @@ def unwrap(self, compound='fragments', reference='com', inplace=True):
             unwrapped compounds will be shifted so that their individual
             reference point lies within the primary unit cell.
             If ``None``, no such shift is performed.
+        reference_point: coordinate, option
+            position where molecule should be unwrapped
```

Comment:
What is the behaviour if `reference_point` is `None` (default)?

Reply:
Also, how does this option behave relative to the `reference` one? If `reference` is `None` but I provide a `reference_point`, what shall I expect?

Maybe `reference` and `reference_point` should be merged so that `reference` can accept a position vector? Or maybe I do not understand how the two options work together.